src/components/Card/Card.tsx

```diff
@@ -11,18 +12,41 @@ function Card({
   flip: boolean;
   onClick: () => void;
 }) {
+  const ref = React.useRef<HTMLDivElement>(document.createElement("div"));
+
+  React.useEffect(() => {
+    let entryPoint = [0, 0];
+    const track = (moveEvent: MouseEvent) => {
```

Comment:
maybe `moveEvent  > mouseEvent`? Usually it's just `event`/`ev`/`e`, depending on the developer :D